connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/RestClient.java

```diff
@@ -139,6 +139,9 @@
                         Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(),
                         "Unexpected status code when handling forwarded request: " + responseCode);
             }
+        } catch(ConnectRestException e) {
```

Comment:
Looks like there are some checkstyle violations in the commit.

It's probably choking on not having a space between the `catch` and open parenthesis.

```suggestion
        } catch (ConnectRestException e) {
```